extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/ExceptionMapperRecorder.java

```diff
@@ -17,4 +17,15 @@ public void setServlets(Map<String, List<String>> servletToMapping) {
         NotFoundExceptionMapper.servlets(servletToMapping);
     }
 
+    /**
+     * Uses to register the paths of classes that are not annotated with JAX-RS annotations (like Spring Controllers for
+     * example)
+     * 
+     * @param nonJaxRsPaths A that contains the class name as a key and class that contains the base path and a map that
+     *        contains the method name to path as a value
+     */
+    public void setNonJaxRsPaths(Map<String, NonJaxRsClassMappings> nonJaxRsPaths) {
```

Comment:
Actually, same here: I'd use a variable name that makes the javadoc non-required.

Reply:
:+1:

Reply:
Any better?